### ISSUE TITLE
Error when dragging to start of list

### ISSUE THREAD

**Issue by lawvs** [toeverything/blocksuite#564]
An error occurs when dragging text to the beginning in the list, it also break format quick bar

```
Uncaught Error: val does not exist
    at Module.assertExists (utils.ts:24:11)
    at assertExists (std.ts:6:9)
    at getFormat (container-operations.ts:189:5)
    at FormatQuickBar.connectedCallback (format-bar-node.ts:77:19)
    at showFormatQuickBar (index.ts:105:13)
    at DefaultSelectionManager._showFormatQuickBar (selection-manager.ts:323:7)
    at DefaultSelectionManager._onContainerDragEnd (selection-manager.ts:312:10)
    at HTMLDocument.mouseUpHandler (gesture.ts:191:7)
assertExists @ utils.ts:24
assertExists @ std.ts:6
getFormat @ container-operations.ts:189
connectedCallback @ format-bar-node.ts:77
showFormatQuickBar @ index.ts:105
_showFormatQuickBar @ selection-manager.ts:323
DefaultSelectionManager._onContainerDragEnd @ selection-manager.ts:312
mouseUpHandler @ gesture.ts:191

format-bar-node.ts:164 Failed to render format-quick-bar! page not found! [ListBlockModel] null
render @ format-bar-node.ts:164
update @ lit-element.ts:160
performUpdate @ reactive-element.ts:1327
scheduleUpdate @ reactive-element.ts:1259
__enqueueUpdate @ reactive-element.ts:1231
await in __enqueueUpdate (async)
requestUpdate @ reactive-element.ts:1206
_initialize @ reactive-element.ts:944
ReactiveElement @ reactive-element.ts:927
LitElement @ lit-element.ts:115
FormatQuickBar @ format-bar-node.ts:26
showFormatQuickBar @ index.ts:41
_showFormatQuickBar @ selection-manager.ts:323
DefaultSelectionManager._onContainerDragEnd @ selection-manager.ts:312
mouseUpHandler @ gesture.ts:191
index.ts:60 

Uncaught (in promise) TypeError: Cannot read properties of null (reading 'getBoundingClientRect')
    at index.ts:60:44
    at std.ts:106:10
    at showFormatQuickBar (index.ts:108:3)
(anonymous) @ index.ts:60
(anonymous) @ std.ts:106
showFormatQuickBar @ index.ts:108
await in showFormatQuickBar (async)
_showFormatQuickBar @ selection-manager.ts:323
DefaultSelectionManager._onContainerDragEnd @ selection-manager.ts:312
mouseUpHandler @ gesture.ts:191
```

https://user-images.githubusercontent.com/18554747/210947663-49276f3e-6910-4213-9240-fad22a151cff.mov

**Comment by himself65** [comment]
This also happened in https://github.com/toeverything/blocksuite/commit/f3fd73ffe8c0dc825af915cf156145b0f173598e

**Comment by doodlewind** [comment]
Closed by #845
